db/blob/blob_source.h

```diff
@@ -83,6 +113,11 @@ class BlobSource {
 
   // A cache to store uncompressed blobs.
   std::shared_ptr<Cache> blob_cache_;
+
+  // Write buffer manager helps users control the total memory used by memtables
+  // across multiple column families and/or DB instances.
+  // https://github.com/facebook/rocksdb/wiki/Write-Buffer-Manager
+  std::shared_ptr<WriteBufferManager> write_buffer_manager_;
```

Comment:
Not sure why we need `write_buffer_manager_` here

Reply:
I want to adjust the dummy entry size when the blob cache usage is changing.

Reply:
Let's take the discussed design